Sources/Yams/Representer.swift

```diff
@@ -14,6 +14,17 @@ import Foundation
 import ucrt
 #endif
 
+#if os(iOS) || os(macOS) || os(watchOS) || os(tvOS)
+import Darwin
+fileprivate let cpow: (_: Double, _: Double) -> Double = Darwin.pow
+#elseif os(Windows)
+import ucrt
+fileprivate let cpow: (_: Double, _: Double) -> Double = ucrt.pow
+#else
+import Glibc
+fileprivate let cpow: (_: Double, _: Double) -> Double = Glibc.pow
+#endif
```

Comment:
Can this be simplified to this?

```swift
#if canImport(Darwin)
private let cpow: (_: Double, _: Double) -> Double = Darwin.pow
#elseif canImport(ucrt)
private let cpow: (_: Double, _: Double) -> Double = ucrt.pow
#elseif canImport(Glibc)
private let cpow: (_: Double, _: Double) -> Double = Glibc.pow
#else
#error("Unsupported platform")
#endif
```

Reply:
I don't think that is really a simplification - it's different semantics.  If I were to have a C module or a Swift module with the name `Darwin` on Windows, this would cause a build failure.  I can certainly change it to that if you feel strongly though.

Reply:
I don't feel strongly, and see the risk of name collisions, although I would advise against naming new modules Darwin for a slew of reasons. I consider it a simplification in the sense that if new platforms were introduced (e.g. `os(HomePod)`) we wouldn't have to touch this code.

Reply:
Sure, but I still think that introduction of an `os(Darwin)` or `vendor(Apple)` is better.  However, that's not the current reality.  I'll update this in a few minutes.

Reply:
No need to update, I'll merge as-is.